vncdotool/client.py

```diff
@@ -251,11 +251,11 @@ def mouseUp(self: TClient, button: int) -> TClient:
 
         return self
 
-    def captureScreen(self, filename: str, incremental: bool = False) -> Deferred:
+    def captureScreen(self, filename: str, incremental: bool = False, format=None) -> Deferred:
         """ Save the current display to filename
         """
         log.debug('captureScreen %s', filename)
-        return self._capture(filename, incremental)
+        return self._capture(filename, incremental, format=None)
```

Comment:
This looks wrong: You probably want to pass through the format here:
```suggestion
        return self._capture(filename, incremental, format=format)
```